bio/mehari/annotate-seqvars/meta.yaml

```
@@ -0,0 +1,37 @@
+name: mehari annotate seqvars
+description: Annotate variant calls with mehari.
+url: https://github.com/varfish-org/mehari
+input:
+  calls: Path to variants (VCF / VCF.gz / BCF)
+  reference_fasta: Path to reference FASTA (uncompressed). Must have an accompanying index.fasta.fai.
```

Maybe just specify FAI as an input 
```suggestion
  ref: Path to reference FASTA (uncompressed).
  fai: Path to refernce FAI file.
```